aptos-move/framework/supra-framework/sources/pbo_delegation_pool.move

```diff
@@ -3228,8 +3237,9 @@ module supra_framework::pbo_delegation_pool {
         );
     }
 
+    // The test case abort because the amount of stake is less than the minimum amount of stake
     #[test(supra_framework = @supra_framework, validator = @0x123)]
-    #[expected_failure(abort_code = 0x10008, location = Self)]
+    #[expected_failure(abort_code = 65574, location = Self)]
```

Comment:
Why this error code change is needed? If required make another test case which fails at the given error code.

Reply:
Ah, I noticed we have the another check for minimal value in the pool at line 1526. That's why the location report changes. I think the new assertion supersede the original one. 

@sjoshisupra Do you want to keep both or I can remove the original one?